README.md

```diff
@@ -17,7 +17,7 @@ A Clojure-inspired atom is placed above the cursor composition, and is the only
 ```js
 const data = Immutable.fromJS({a: 1, b: 2});
 
-const cursor = Cursor.from(data, (prevValue, nextValue) => {
+const cursor = Cursor.from(data, (nextValue, prevValue, keyPath) => {
   console.log(newData)
```

Comment:
Due to my own stupidity, we're `console.log`'ing an undeclared variable here. Are you able to sneak in a new commit whilst you're touching this file, to update this to `nextValue`?

I'm actually thinking whether we should also include `prevValue` and `keyPath` in the `console.log` too for explicitness. What do you think?